plugin-build/plugin/src/main/java/io/sentry/kotlin/multiplatform/gradle/plugin/SentryPlugin.kt

```diff
@@ -0,0 +1,231 @@
+package io.sentry.kotlin.multiplatform.gradle.plugin
+
+import org.gradle.api.GradleException
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.plugins.ExtensionAware
+import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
+import org.jetbrains.kotlin.gradle.plugin.cocoapods.CocoapodsExtension
+import org.jetbrains.kotlin.gradle.plugin.cocoapods.KotlinCocoapodsPlugin
+import org.jetbrains.kotlin.gradle.plugin.mpp.Framework
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+import org.jetbrains.kotlin.gradle.plugin.mpp.TestExecutable
+import java.io.ByteArrayOutputStream
+import java.io.File
+
+internal const val SENTRY_EXTENSION_NAME = "sentry"
```

Comment:
`m`: I'm wondering if we should consider the case where both this plugin and SAGP are used. In that case there will probably a conflict of two extensions with the same name, so perhaps we should consider something different here, like `sentryKmp` or `sentryMultiplatform`

Reply:
yeah definitely